### PR TITLE
Adding Adel to participants

### PR DESCRIPTION
**Pull request by adelsmee**
THIS IS A PULL REQUEST TEMPLATE! 
It helps you complete a pull request. Please follow the instructions below.

## Description

I was here and I participated and I'd like to be added to participants, please.

## Changes made in this pull request
Me, added, to participants...

*Once your pull request is opened, please notify Josh.*

### CODE REVIEW

File: participants/adel.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# ${NAME}
```
Comment (lorderikir) [inline participants/adel.md]: Please replace `${NAME}` with your name

Comment (lorderikir) [inline participants/adel.md]:
```suggestion
# ${Adel}
```

Comment (hugomd) [reply participants/adel.md]: Shouldn't this just be
```suggestion
# Adel
```

Comment (lorderikir) [reply participants/adel.md]: Yeeah

**Pull request merged**